schema/release-schema.json

```
@@ -2639,6 +2657,30 @@
           "format": "uri"
         }
       }
+    },
+    "SimpleIdentifier": {
+      "title": "Simple identifier",
+      "description": "An unambiguous reference to a resource within a given context.",
+      "type": "object",
+      "properties": {
+        "scheme": {
+          "title": "Scheme",
+          "description": "The list, register or system from which identifiers are taken.",
```

Change word agreement, since this is in the context of one identifier.

```suggestion
          "description": "The list, register or system from which the identifier is taken.",
```